src/ynETH.sol

```diff
@@ -124,6 +132,20 @@ contract ynETH is IynETH, ynBase, IYnETHEvents {
         emit Deposit(msg.sender, receiver, assets, shares, totalDepositedInPool);
     }
 
+    /// @notice Calculates the amount of shares to be minted for a given deposit.
+    /// @param assets The amount of assets to be deposited.
+    /// @return The amount of shares to be minted.
+    function previewDeposit(uint256 assets) public view virtual returns (uint256) {
```

Comment:
huh, but this and the function below are exactly the same apart from the function name?!

Reply:
followed the erc4626 convention which has both